### PR TITLE
BaSP-TG-06: Admin Endpoints (GetAll and Create)

### PR DESCRIPTION
**Pull request by PilarFernandezC**
- Create: creates an admin with the data completed in the body
- GetAll: shows a list of all the admins in the data base

### CODE REVIEW

File: src/controllers/admins.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    });
+
+    const result = await admin.save();
+    return res.status(201).json({
+      message: 'Admin created successfully',
+      data: result,
+    });
+  } catch (error) {
+    return res.status(400).json({
+      message: 'An error ocurred',
```
Comment (ignacioaristo) [inline src/controllers/admins.js:19]: we can handle better this message, we can do somethin like : `message: 'An error ocurred': ${error}`

File: src/controllers/admins.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      error,
```
Comment (ignacioaristo) [inline src/controllers/admins.js:20]: i think this prop is unnecessary.

File: src/controllers/admins.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+const getAllAdmins = async (req, res) => {
+  try {
+    const admins = await Admins.find();
+    return res.status(200).json({
+      message: 'Admin found',
+      data: admins,
+    });
+  } catch (error) {
+    return res.json({
```
Comment (ignacioaristo) [inline src/controllers/admins.js:33]: status code is missing.

File: src/controllers/admins.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      message: 'An error occurred',
```
Comment (ignacioaristo) [inline src/controllers/admins.js:34]: same here

File: src/controllers/admins.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      error,
```
Comment (ignacioaristo) [inline src/controllers/admins.js:35]: and here

**Review by ignacioaristo — CHANGES_REQUESTED** [review]
The postman collection is missing, we have to add the file.

File: src/controllers/admins.js (hunk 1 above)
Comment (PilarFernandezC) [reply src/controllers/admins.js]: done

File: src/controllers/admins.js (hunk 3 above)
Comment (PilarFernandezC) [reply src/controllers/admins.js:37]: I added it with an if, I don't know if it's ok

File: src/controllers/admins.js (hunk 4 above)
Comment (PilarFernandezC) [reply src/controllers/admins.js]: done

File: src/controllers/admins.js (hunk 5 above)
Comment (PilarFernandezC) [reply src/controllers/admins.js]: done

**Review by ignacioaristo — APPROVED** [review]

**Review by paurinaldi — APPROVED** [review]

**Pull request merged**